didcomm/did_doc/did_doc.py

```diff
@@ -1,50 +1,28 @@
 from __future__ import annotations
 
-import attr
-from dataclasses import dataclass
-from typing import List, Optional
-
-from didcomm.common.types import (
-    DID_URL,
-    DID,
-    VerificationMaterial,
-    VerificationMethodType,
-)
-from didcomm.common.utils import search_first_in_iterable
+from typing import Optional
 
+from pydid import DIDCommService
+from pydid import VerificationMethod
```

Comment:
By convention, these should be included in an `__all__` array if we intend for them to be used outside of this module -- if nothing else, to express the intent that they may be used outside of this module.

Reply:
They are already included (in the main branch). Now I just replaced the implementation with the one from pydid.